app/helpers/admin/taggable_content_helper.rb

```diff
@@ -144,7 +144,7 @@ def taggable_document_collection_groups_container
   # organisation and its ID.
   def taggable_worldwide_organisations_container
     Rails.cache.fetch(taggable_worldwide_organisations_cache_digest, expires_in: 1.day) do
-      WorldwideOrganisation.with_translations.map {|wo| [wo.name, wo.id] }
+      WorldwideOrganisation.with_translations(I18n.locale).map {|wo| [wo.name, wo.id] }
```

Comment:
We only have an english admin so I don't think it would work with any other locale. But have updated for you.